python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -626,6 +637,15 @@ def _add_min_workers_nodes(
     return node_resources, node_type_counts, total_nodes_to_add_dict
 
 
+def transform_node_types(
```

Comment:
Name of function is vague.
Could use a doc string and an explanatory comment where it's called.